sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -70,6 +71,96 @@ describe("New session token", function () {
   });
 });
 
+describe.only("Integrated Cache Staleness", async function (this: Suite) {
```

Comment:
Don't forget to remove `.only` otherwise every other test is going to be skipped